### PR TITLE
Add new OMs of the month

### PR DESCRIPTION
**Pull request by Spandamn**

### CODE REVIEW

**Comment by Spandamn** [comment]
This is gonna take many commits :(
srry this is my first big pr

File: config/formats.js [hunk 1]
```diff
@@ -238,39 +238,70 @@ exports.Formats = [
...
+				"galvanize": true,
+				"parentalbond": true,
+				"pixilate": true,
+				"refrigerate": true,
+				"sheerforce": true,
+				"slowstart": true,
+				"truant": true,
+				"unburden": true,
+				"zenmode": true
+			};
```
Comment (TheImmortal) [inline config/formats.js]: Use ``1``. Also can you put it on a single line.

Comment (Spandamn) [inline config/formats.js]: That is what im doing rn :)

Comment (Spandamn) [inline config/formats.js]: also im making it an array

File: config/formats.js [hunk 2]
```diff
@@ -238,39 +238,70 @@ exports.Formats = [
...
-		ruleset: ['Pokemon', 'Standard', 'Team Preview', 'Baton Pass Clause'],
-		banlist: ['Ignore STAB Moves',
-			'Aegislash', 'Arceus', 'Blaziken', 'Darkrai', 'Deoxys', 'Dialga', 'Giratina', 'Groudon', 'Ho-Oh', 'Kyogre', 'Kyurem-White',
-			'Landorus-Base', 'Lugia', 'Lunala', 'Mewtwo', 'Palkia', 'Rayquaza', 'Reshiram', 'Shaymin-Sky', 'Solgaleo', 'Xerneas', 'Yveltal', 'Zekrom',
-			'Power Construct', 'Shadow Tag', 'Aerodactylite', 'Gengarite', 'Kangaskhanite', "King's Rock", 'Lucarionite', 'Metagrossite', 'Razor Fang', 'Salamencite',
+			"Pokémon have all their natural abilities at the same time.",
+			"&bullet; <a href=\"http://www.smogon.com/forums/threads/pok%C3%A9bilities.3510241/\">Pokébilities</a>"
 		],
+		
+		mod: 'pokebilities',
```
Comment (TheImmortal) [inline config/formats.js]: Where is the mod

File: config/formats.js [hunk 3]
```diff
@@ -238,39 +238,70 @@ exports.Formats = [
...
-		],
-
-		mod: 'gen7',
-		ruleset: ['Pokemon', 'Standard', 'Team Preview', 'Baton Pass Clause'],
-		banlist: ['Ignore STAB Moves',
-			'Aegislash', 'Arceus', 'Blaziken', 'Darkrai', 'Deoxys', 'Dialga', 'Giratina', 'Groudon', 'Ho-Oh', 'Kyogre', 'Kyurem-White',
-			'Landorus-Base', 'Lugia', 'Lunala', 'Mewtwo', 'Palkia', 'Rayquaza', 'Reshiram', 'Shaymin-Sky', 'Solgaleo', 'Xerneas', 'Yveltal', 'Zekrom',
-			'Power Construct', 'Shadow Tag', 'Aerodactylite', 'Gengarite', 'Kangaskhanite', "King's Rock", 'Lucarionite', 'Metagrossite', 'Razor Fang', 'Salamencite',
+			"Pokémon have all their natural abilities at the same time.",
+			"&bullet; <a href=\"http://www.smogon.com/forums/threads/pok%C3%A9bilities.3510241/\">Pokébilities</a>"
```
Comment (TheImmortal) [inline config/formats.js]: Remove the thread name, so it should be /3510241/

Also might as well make both links https

File: config/formats.js (hunk 2 above)
Comment (Spandamn) [inline config/formats.js]: creating

File: config/formats.js [hunk 4]
```diff
@@ -238,39 +238,70 @@ exports.Formats = [
...
-			"&bullet; <a href=\"https://www.smogon.com/forums/threads/3587949/\">STABmons</a>",
-		],
-
-		mod: 'gen7',
-		ruleset: ['Pokemon', 'Standard', 'Team Preview', 'Baton Pass Clause'],
-		banlist: ['Ignore STAB Moves',
-			'Aegislash', 'Arceus', 'Blaziken', 'Darkrai', 'Deoxys', 'Dialga', 'Giratina', 'Groudon', 'Ho-Oh', 'Kyogre', 'Kyurem-White',
-			'Landorus-Base', 'Lugia', 'Lunala', 'Mewtwo', 'Palkia', 'Rayquaza', 'Reshiram', 'Shaymin-Sky', 'Solgaleo', 'Xerneas', 'Yveltal', 'Zekrom',
-			'Power Construct', 'Shadow Tag', 'Aerodactylite', 'Gengarite', 'Kangaskhanite', "King's Rock", 'Lucarionite', 'Metagrossite', 'Razor Fang', 'Salamencite',
+			"Pokémon have all their natural abilities at the same time.",
```
Comment (TheImmortal) [inline config/formats.js]: Pok&eacute;mon

File: config/formats.js (hunk 3 above)
Comment (Spandamn) [inline config/formats.js]: True

File: config/formats.js (hunk 4 above)
Comment (Spandamn) [inline config/formats.js]: hmm ok

File: config/formats.js [hunk 5]
```diff
@@ -238,39 +238,70 @@ exports.Formats = [
...
+			}
+		},
 	},
 	{
-		name: "[Gen 7] Middle Cup",
+		name: "[Gen 7] 350 Cup",
 		desc: [
-			"Only middle evolutions are allowed.",
-			"&bullet; <a href=\"https://www.smogon.com/forums/threads/3588047/\">Middle Cup</a>",
+			"Pokémon having a base stat total 350 or below get their stats doubled..",
```
Comment (TheImmortal) [inline config/formats.js]: ``Pok&eacute;mon``

Also change the desc to: Pokémon with a base stat total of 350 or lower get their stats doubled.

Also there's two periods there

Comment (Spandamn) [inline config/formats.js]: riggt

**Comment by TheImmortal** [comment]
I was hoping you would do this. I need someone to take over from me one day! :)

**Comment by TheImmortal** [comment]
Also fix all the Travis errors

File: config/formats.js [hunk 6]
```diff
@@ -238,39 +238,56 @@ exports.Formats = [
...
+			}
+		},
 	},
 	{
-		name: "[Gen 7] Middle Cup",
+		name: "[Gen 7] 350 Cup",
 		desc: [
-			"Only middle evolutions are allowed.",
-			"&bullet; <a href=\"https://www.smogon.com/forums/threads/3588047/\">Middle Cup</a>",
+			"Pokémon with a base stat total of 350 or lower get their stats doubled.",
```
Comment (TheImmortal) [inline config/formats.js]: ``Pok&eacute;mon``

Comment (Spandamn) [inline config/formats.js]: oh forgot that one

File: config/formats.js [hunk 7]
```diff
@@ -238,39 +238,56 @@ exports.Formats = [
...
-		banlist: ['Eviolite', 'Light Ball', 'Baton Pass'],
-		onValidateSet: function (set) {
-			let template = this.getTemplate(set.species || set.name);
-			if (!template.prevo || !template.nfe) {
-				return [set.species + " is not the middle Pokémon in an evolution chain."];
-			}
-		},
+		mod: '350cup',
+		ruleset: ['[Gen 7] Pokebank Ubers'],
+		banlist: ['Eviolite', 'Light Ball', 'Deep Sea Tooth', 'Eevium Z'],
```
Comment (TheImmortal) [inline config/formats.js]: I like to sort things alphabetically

File: mods/pokebillities/scripts.js [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+						let abe = target.innates[this.random(target.innates.length)];
+						let bannedAbilities = {
+							flowergift: 1,
+							forecast: 1,
+							illusion: 1,
+							imposter: 1,
+							multitype: 1,
+							stancechange: 1,
+							trace: 1,
+							otherzenmode: 1
```
Comment (TheImmortal) [inline mods/pokebillities/scripts.js]: ???

Comment (Spandamn) [inline mods/pokebillities/scripts.js]: I copy pasted that from Trace's code
well, time to update it

File: mods/pokebilities/scripts.js [hunk 9]
```diff
@@ -27,17 +27,8 @@ exports.BattleScripts = {
...
-							forecast: 1,
-							illusion: 1,
-							imposter: 1,
-							multitype: 1,
-							stancechange: 1,
-							trace: 1,
-							otherzenmode: 1
-						};
+						abe = target.innates[this.random(target.innates.length)];
+						let bannedAbilities = {comatose:1, disguise:1, flowergift:1, forecast:1, illusion:1, imposter:1, multitype:1, schooling:1, stancechange:1, trace:1, otherzenmode:1};
```
Comment (TheImmortal) [inline mods/pokebilities/scripts.js]: I was referring to the ``otherzenmode``.

File: mods/pokebilities/scripts.js [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+					let possibleTargets = [];
+					for (let i = 0; i < pokemon.side.foe.active.length; i++) {
+						if (pokemon.side.foe.active[i] && !pokemon.side.foe.active[i].fainted) possibleTargets.push(pokemon.side.foe.active[i]);
+					}
+					while (possibleTargets.length) {
+						let rand = 0;
+						if (possibleTargets.length > 1) rand = this.random(possibleTargets.length);
+						let target = possibleTargets[rand],
+							abe = target.innates[this.random(target.innates.length)];
+						let bannedAbilities = {comatose:1, disguise:1, flowergift:1, forecast:1, illusion:1, imposter:1, multitype:1, schooling:1, stancechange:1, trace:1, otherzenmode:1};
```
Comment (TheImmortal) [inline mods/pokebilities/scripts.js]: What is ``otherzenmode``

Comment (Spandamn) [inline mods/pokebilities/scripts.js]: abilities which also have a status are renamed to "other"+<ability name>

**Comment by Spandamn** [comment]
:)

**Comment by TheImmortal** [comment]
Did you test it, this latest version of the code

**Comment by Spandamn** [comment]
Srry got dcd, yes the code works

**Pull request merged**